knack/index.js

```diff
@@ -531,28 +531,22 @@ $(document).on("knack-view-render.view_2609", function(event, scene) {
   $("#kn-map-field_3300").hide(); // Remove map from Location Details
 });
 
-// IN PROGRESS: Mike's code for 51_lat_lon_layout
-$(document).on("knack-view-render.view_2607", function(event, scene) {
-  $header = $("#view_2607 > div.view-header");
-  $(document).on("knack-view-render.view_2586", function(event, scene) {
-    $("#view_2586").append($header);
-  });
-  $form = $("#view_2607");
+// Overlay Latitude/Longitude fields and button on map
+$(document).on("knack-scene-render.scene_1028", function(event, scene) {
```

Comment:
Looks like we're using the `view-render` event in addition to the `scene-render`. Does the code for each of these things need to be in separate places?

Reply:
The jQuery in the view-render does need to be separate from the scene-render since that view renders after the submission of the lat/lon fields without a full page reload (the jQuery inside the scene-render handles all of the changes for full page reload). There may be a way to combine these, but it gets tricky with the form submission.